assets/js/src/subscribers/importExport/import.jsx

```diff
@@ -157,17 +157,26 @@ jQuery(document).ready(() => {
             .replace('%1$s', `<strong>${subscribers.duplicate.length}</strong>`)
             .replace('%2$s', subscribers.duplicate.join(', '))
           : null,
+        role: (subscribers.role.length)
+          ? MailPoet.I18n.t('importNoticeRoleBased')
+            .replace('%1$s', `<strong>${subscribers.role.length.toLocaleString()}</strong>`)
+            .replace('%2$s', subscribers.role.join(', '))
+            .replace(
+              /\[link](.+)\[\/link]/,
+              '<a href="https://kb.mailpoet.com/article/270-role-based-email-addresses-are-not-allowed" target="_blank" rel="noopener noreferrer">$1</a>'
+            )
+          : null,
       };
       jQuery('#subscribers_data_parse_results').html(
         subscribersDataParseResultsTemplate(importResults)
       );
     }
 
     jQuery('.mailpoet_subscribers_data_parse_results_details_show')
-      .click((event) => {
+      .click(function detailsClick() {
         const details = jQuery('.mailpoet_subscribers_data_parse_results_details');
         jQuery(details).toggle();
```

Comment:
Not part of this issue, but `jQuery(details).toggle();` can be also written as `details.toggle();` - the `jQuery` is unnecessary.